eth_typing/abi.py

```diff
@@ -143,23 +143,24 @@ class ABIError(TypedDict):
 
 ABICallable = Union[ABIFunction, ABIConstructor, ABIFallback, ABIReceive]
 
+ABIElement = Union[ABICallable, ABIEvent, ABIError]
+"""Base type for `ABIFunction` and `ABIEvent` types."""
```

Comment:
If I read just the docstring here and didn't look at the definition for this type, I'd assume that "Base type" would mean that it is a base class that the referenced types all inherit from, such that if I used `isinstance(ABIElement)` on any one of the mentioned type instances it should return `True`. That isn't the case with these relationships. I think describing it as a `Union` would make more sense. We should also add `ABIError` to the docstring.

```suggestion
"""A `Union` type consisting of `ABIFunction`, `ABIEvent`, and `ABIError` types."""
```